### PR TITLE
Albatross: write timeout change and minor code refactoring

### PR DESCRIPTION
**Pull request by burmancomp**
### Link to issue number:
none
### Summary of the issue:
Driver used 0 for write_timeout (non-blocking). If display was updated frequently (for example pressing and holding arrow key down in long document), display updating continued some time after for example arrow key was not pressed anymore. Using write_timeout 0.2 seconds (quite typical value with other braille drivers) resolves this.
### Description of user facing changes
Braille shows pretty quickly valid information after situations where braille line was updated frequently (for example when pressing and holding arrow key down in long document).
### Description of development approach
In addition to write_timeout change, there is a change how display is updated after reconnection and exit from internal menu. Using braille.handler._displayWithCursor function, driver code can be reduced.
### Testing strategy:
Tested with Albatross 80 model.
### Known issues with pull request:
none known
### Change log entries:
New features
Changes
Bug fixes
For Developers

### Code Review Checklist:

- [x] Pull Request description:
  - description is up to date
  - change log entries
- [x] Testing:
  - Unit tests
  - System (end to end) tests
  - Manual testing
- [x] API is compatible with existing add-ons.
- [x] Documentation:
  - User Documentation
  - Developer / Technical Documentation
  - Context sensitive help for GUI changes
- [x] UX of all users considered:
  - Speech 
  - Braille
  - Low Vision
  - Different web browsers
  - Localization in other languages / culture than English
- [x] Security precautions taken.

### CODE REVIEW

File: source/brailleDisplayDrivers/albatross/constants.py [hunk 1]
```diff
@@ -53,9 +53,9 @@
 """
 
 READ_TIMEOUT = 0.2
-WRITE_TIMEOUT = 0
+WRITE_TIMEOUT = 0.2
 SLEEP_TIMEOUT = 0.2
-"""How long to sleep between port init or open retries."""
+"""How long to sleep when port open or I/O buffers reset fails."""
```
Comment (seanbudd) [inline source/brailleDisplayDrivers/albatross/constants.py:58]: Grammar fix

```suggestion
"""How long to sleep when port opens or I/O buffers reset fails."""
```

File: source/brailleDisplayDrivers/albatross/driver.py [hunk 2]
```diff
@@ -114,14 +111,11 @@ def __init__(self, port="Auto"):
 		self._exitEvent = Event()
 		# Thread for read
 		self._handleRead = None
-		# Display function is called manually when display is switched back on
-		# or exited from internal menu. Ensuring data integrity.
-		self._displayLock = Lock()
```
Comment (seanbudd) [inline source/brailleDisplayDrivers/albatross/driver.py:119]: could you explain a bit why the lock was removed? It's somewhat concerning

Comment (burmancomp) [reply source/brailleDisplayDrivers/albatross/driver.py:119]: When reconnected or exited from internal menu braille.handler._displayWithCursor() is executed so there should be no reason to execute display function manually. My assumption is that possible conflicts are thus resolved elsewhere and thus lock is redundant.

Comment (seanbudd) [reply source/brailleDisplayDrivers/albatross/driver.py:119]: The display function should be  able to handle being executed from other threads. While I think current contexts may be safe for the driver, it's better to keep the lock.

**Review by seanbudd — APPROVED** [review]
Thanks @burmancomp

**Pull request merged**